### ISSUE TITLE
3.7 support

### ISSUE THREAD

**Issue by kyle-schleich** [Jc2k/hpfeeds3#5]
What is the plan on adding support for 3.7?

I know currently at least this breaks with async clients right now.
(in asyncio/client.py)
```
async def __aiter__(self): 
    return self
```

**Comment by Jc2k** [comment]
Hey @kyle-schleich - i've added a 3.7 builder on travis + i'll see if I can repro your issue. Don't suppose you have a traceback of it blowing up do you?

It looks like this might be the fix:

```
diff --git a/hpfeeds/asyncio/client.py b/hpfeeds/asyncio/client.py
index afb6588..01e516b 100644
--- a/hpfeeds/asyncio/client.py
+++ b/hpfeeds/asyncio/client.py
@@ -155,7 +155,7 @@ class ClientSession(object):
     async def __aexit__(self, exc_type, exc_val, exc_tb):
         await self.close()
 
-    async def __aiter__(self):
+    def __aiter__(self):
         return self
 
     async def __anext__(self):
```

But i need to test it and figure out how to not break 3.6 support.

**Comment by Jc2k** [comment]
This should be fixed in 0.9.8, so closing this ticket.
